src/CMakeLists.txt

```diff
@@ -543,6 +544,18 @@ if(BUILD_CLIENT)
 			${client_LIBS}
 		)
 	endif()
+	if(ENABLE_GLES)
+		target_link_libraries(
+			${PROJECT_NAME}
+			${OPENGLES2_LIBRARIES}
+			${EGL_LIBRARIES}
+		)
+	else()
+		target_link_libraries(
+			${PROJECT_NAME}
+			${OPENGL_LIBRARIES}
```

Comment:
OpenGL could theoretically still be used on OGLES2 builds. How about including both, if they're provided?

Reply:
The problem is that `ENABLE_GLES` changes some parts of the code, whether you use GLES at runtime or not.
It'd probably still work, but I don't really want to enable this until Minetest (one day) properly has coexisting GL/GLES support.